keyboards/kudox/rev1/rev1.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include "../kudox.h"
+
+#include "quantum.h"
+
+
+#ifdef USE_I2C
```

Comment:
Pretty sure this whole block is unnecessary, as split common should be setting all of this up (I think).